src/course-unit/clipboard/paste-component/constants.js

```diff
@@ -0,0 +1,12 @@
+import PropTypes from 'prop-types';
+
+export const clipboardProps = {
```

Comment:
[question] What do you think about renaming it to `clipboardPropsTypes`?

Reply:
Good idea